docs/how-to-create-a-new-release.rst

```diff
@@ -160,11 +160,26 @@ Uploading to PyPI
 
 .. _twine: https://pypi.python.org/pypi/twine
 
+Updating the CEA Electron interface
+-----------------------------------
+
+For the installer to be able to pick up the newest version of the CEA Electron interface, you need to build it,
+create a release and attach a ``win-unpacked.7z`` to the release. Here are the steps:
+
+- pull the newest version of the ``cea-electron`` repository
+- open CEA Console, navigate to the GitHub repo of the ``cea-electron`` repository
+- type ``yarn dist:dir``, wait for the command to complete
+- the subfolder ``dist`` will contain a folder ``win-unpacked``
+- compress the folder ``win-unpacked`` to ``win-unpacked.7z``
+- open https://github.com/architecture-building-systems/cea-electron/releases in the browser
+- Draft a new release, use a tag corresponding to the CEA version number (e.g. ``v2.23``)
```

Comment:
Hmmm. I feel that the version number for the interface should not follow CEA since it would not make sense if the minor version suddenly jumps if it has not changed for a while compared to CEA, of course that would be more troublesome. I dont know, what do you think?

Reply:
I think the version numbers correspond to the milestones / sprints. `cea-electron` doesn't have it's own release cycle, right? In my mind, it's closely linked to the CEA lifecycle (the cea-electron repository can't really be used outside of the CEA context at all anyway). So, I don't really see much of a problem.

If it really bothers you, can you recommend a way to name the tags, so that they're not cea-electron versions, but denote that they _refer_ to a CEA release?

I really think it's easier if the releases of cea-electron are based on the CEA release cycle as the two are so closely linked. _theoretically_ the situation could arise where `cea-electron` is not updated during a sprint. _practically_, I think the opposite is more likely given the current priorities.

Reply:
In my opinion, the releases of cea-electron (now called CityEnergyAnalyst-dashboard) can be separate. The reason is that everytime an update of the interface exists (after solving a bug) we need to make a build and publish it. At least that is how it works now.

We are facing this now with Reynold, where I checked this PR, found several bugs that needed to be fixed and tested. But to do that. We need to make another build, I have to download the build, replace files and test. that is just a long process.

Ideally, we create a way to automatically update the interface once a new build is available. Still we will need that build to be labeled somewhere.

Maybe I am missing something? Mind a chat so we make this clear? It would save some time of bouncing the ball back..

Reply:
I'm on vacation all next week, so a chat won't be possible until after that. Here some thoughts:

- CityEnergyAnalyst-dashboard... we missed a chance to re-brand what we call the whole interface as opposed to the plotting stuff. How about CityEnergyAnalyst-GUI?
- The CEA installer picks up the _latest_ published release of the CityEnergyAnalyst-dashboard from GitHub. So you can easily publish bugfixes by doing `v2.24.1`, `v2.24.2` etc.
- Even during bug fixing, it might be prudent to keep official releases on the same release schedule as the CEA (bi-weekly) - unless a hotfix is necessary. This creates some stability.
- an auto-update feature for the interface should be an issue: this is a great idea!
- regarding testing: that should work similar to how the development process of CEA works: `git pull` on the `CityEnergyAnalyst-dashboard` repo, type `yarn dev` to start the electron interface. Am I missing something?